tests/nanoemoji_test.py

```diff
@@ -207,9 +219,15 @@ def test_build_cbdt_font():
         "compat_font/config.toml",
     ],
 )
+@pytest.mark.parametrize("use_zopflipng", [True, False])
 @pytest.mark.skipif(RESVG_PATH is None, reason="resvg not installed")
-def test_build_compat_font(config_file):
-    tmp_dir = _run((locate_test_file(config_file),))
+def test_build_compat_font(config_file, use_zopflipng):
```

Comment:
doesn't really demonstrate zopfli'ing actually does anything? Perhaps we could run on/off and assert bitmap is smaller? - technically not guarranteed but we should be able to use a test asset where it's extremely probable

Reply:
good point, i was lazy :)